cnnlabs/cnn-opp-brain-api
# live_feeds: add route for video live feeds

Adding ability to pull live feed metadata from Ptolemy for livefeeds
1-3. Live4 that we use is not in the CMS and does not follow the normal patterns of the live feed videos.

To test locally:
1. `npm run start` (include ENVIRONMENT + PORT)
2. http://localhost:5060/documentation#!/livefeeds/getLivefeeds
3. Click "Try it out"
4. You should see the 3 stream's meta data returned in order from 1-3.

Fixes: CNNETANTWO-79
Reviewed-By: Katie Owen <katherine.owen@turner.com>
